### ISSUE TITLE
index 0 out of bounds for length 0

### ISSUE THREAD

**Issue by FarzyYT** [AllTheMods/ATM-9#472]
I can't play with my friend we both have this error while joining. We play at local network with radmin vpn.

**Comment by oliver-chapman** [comment]
More info than that is needed, is this a dedicated server or local host?

If it's locally hosted, is the port open on your router? (Typically port 25565)
Any error logs also help.

Also, you've posted the same thing on other issues for specific mods, that adds misinformation as it may confuse developers looking at it and seeing your error which probably has nothing to with the mods you commented on.

**Comment by Caprikey** [comment]
I have received this error before over multiple versions of the All The Mods 9 build, as well. 

For me, this typically only occurs when we create a new world. I have been able to resolve it with the following steps: 

1. Verify everyone is on the same build of the modpack. 
2. Have the person who will be acting as the LAN Host/world creator, create the new world. 
3. Once the world has been generated successfully, we have the world creator "Save and Quit". 
4. Then the world creator, relaunch the world. 
5. Then we open to LAN and connect.

**Comment by DLBPointon** [comment]
Hi,
I've had this error across multiple users in the house for a month or so now. 
I've found that a complete clean reinstall of the mod back (or downloading the profile from curseforge) fixes it.

The fix suggested above by Caprikey worked initially for 2 sessions and then stopped... no idea.

From what I can tell updating a profile breaks it, or making duplicates of the broken ( as in one where the LAN doesn't work) profile that you then update to any version, so perhaps there's something in there that isn't updated properly? 

This started occurring on 0.0.65 until 0.1.11 (where I just deleted everything on everyone's computers and reinstalled the pack).

I don't have the modpacking knowledge to figure out anything more. But that's what has worked for us. Just as Stargates get added which is just awesome.
